Velaptor/InternalExtensionMethods.cs

```diff
@@ -29,6 +29,39 @@ namespace Velaptor
     /// </summary>
     internal static class InternalExtensionMethods
     {
+        /// <summary>
+        /// Determines whether this string instance does not start with the specified character.
+        /// </summary>
+        /// <param name="stringToCheck">The string to check.</param>
+        /// <param name="value">The character to compare.</param>
+        /// <returns><c>true</c> if <paramref name="value"/> matches the beginning of this string; otherwise, <c>false</c>.</returns>
+        public static bool DoesNotStartWith(this string stringToCheck, char value) => !stringToCheck.StartsWith(value);
+
+        /// <summary>
+        /// Determines whether this string instance does not start with the specified string.
+        /// </summary>
+        /// <param name="stringToCheck">The string to check.</param>
+        /// <param name="value">The string to compare.</param>
+        /// <returns><c>true</c> if <paramref name="value"/> matches the beginning of this string; otherwise, <c>false</c>.</returns>
+        public static bool DoesNotStartWith(this string stringToCheck, string value) => !stringToCheck.StartsWith(value);
+
+        /// <summary>
+        /// Determines whether the end of this string instance matches the specified character.
+        /// </summary>
+        /// <param name="stringToCheck">The string to check.</param>
+        /// <param name="value">The character to compare to the character at the end of this instance.</param>
+        /// <returns><c>true</c> if <paramref name="value"/> matches the end of this instance; otherwise, <c>false</c>.</returns>
+        public static bool DoesNotEndWith(this string stringToCheck, char value) => !stringToCheck.EndsWith(value);
+
+        /// <summary>
+        /// Determines whether the end of this string instance matches the specified string.
+        /// </summary>
+        /// <param name="stringToCheck">The string to check.</param>
+        /// <param name="value">The string to compare to the character at the end of this instance.</param>
+        /// <returns><c>true</c> if <paramref name="value"/> matches the end of this instance; otherwise, <c>false</c>.</returns>
+        [SuppressMessage("ReSharper", "UnusedMember.Global", Justification = "Kept for future development.")]
+        public static bool DoesNotEndWith(this string stringToCheck, string value) => !stringToCheck.EndsWith(value);
+
         /// <summary>
         /// Returns a value indicating whether the given file or directory path
         /// only contains a root drive path with no directories.
```

Comment:
SS
```suggestion
        /// contains a root drive path with no directories.
```
Not sure if removing ONLY is appropriate.